tests/test_stbt_control_relay.py

```diff
@@ -104,6 +105,8 @@ def preexec_fn():
         os.environ['LISTEN_PID'] = str(os.getpid())
         if fd != 3:
             os.dup2(fd, 3)
+        if sys.version_info.major > 2:
+            os.set_inheritable(3, True)  # pylint:disable=no-member
```

Comment:
I wonder why they didn't backport this in `future`?  It should just call through to set `O_CLOEXEC`.